### DISCUSSION TITLE
Add space between buttons

### DISCUSSION THREAD

**Discussion by Jasminahmad** [quasarframework/quasar#13685] [General - Components / Directives / etc]
<img width="598" alt="Screenshot 2022-06-14 at 10 55 17 AM" src="https://user-images.githubusercontent.com/76095479/173484239-bc93a942-ecbc-4bad-819c-0b4b35896f85.png">

How do I add space in between those two buttons?

**Comment by pdanpdan** [comment]
What picture editor are you using?

**Comment by suleimanas** [comment]
https://quasar.dev/layout/grid/row#example--horizontal-alignment

```
<div class="row justify-between">
... your btns
</div>
```

**Comment by affablelochan** [comment]
Having btn in their own div worked. 

```
     <div>
        <q-btn label="Submit" type="submit" color="primary"/>
      </div>
      <div>
        <q-btn label="Reset" type="reset" color="primary"  />
      </div>
```
![image](https://user-images.githubusercontent.com/5964118/235537204-856d4f6f-6bbd-4f64-b509-73e913a40d26.png)

**Comment by surrealdreams** [comment]
You can also use the q-gutter- classes on a containing div.  q-gutter-sm, q-gutter-md, q-gutter-lg, etc.
